daemons/gptp/linux/src/linux_hal_persist_file.cpp

```diff
@@ -123,7 +123,9 @@ class LinuxGPTPPersistFile : public GPTPPersist {
 
 		bool result = false;
 		if (memoryDataLength > storedDataLength) {
-			ftruncate(persistFD, memoryDataLength);
+			int ret = ftruncate(persistFD, memoryDataLength);
```

Comment:
I don't understand the point of this.  It doesn't change any behavior, so the compiler will most likely not even emit any code for it. I'm all for adding return code checks, but they should at least use the existing log mechanisms to indicate what happened when a non-success return code is found.